src/components/Footer.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <div class="container">
+    <footer class="pt-4 my-md-5 pt-md-5 border-top">
+        <div class="row">
+            <div class="col-12 col-md">
+                <h5 class="font-weight-normal">
+                    <kbd>DominicanWho.Codes</kbd>
+                </h5>
+                <small class="d-block mb-3 text-muted">© 2019</small>
+            </div>
+            <div class="col-6 col-md">
+                <h5>Features</h5>
```

Comment:
Add a `font-weight: 700;` to all the h5 of this section,  they should be bold :P